jetstream/tests/test_external_config.py

```diff
@@ -21,9 +21,6 @@ def test_from_github_repo(self):
         external_configs = ExternalConfigCollection.from_github_repo()
         assert external_configs
 
-        example_conf = external_configs.spec_for_experiment("example_config")
-        assert example_conf is not None
-
```

Comment:
Example configs are getting ignored as of https://github.com/mozilla/jetstream/pull/528, so this test is currently always failing